### PR TITLE
post create post card and comment list add in  Feed

### PR DESCRIPTION
**Pull request by Yatanvesh**

### CODE REVIEW

File: src/components/Profile/CommentList.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        name={data.name}
+        comment={data.comment}
+        like={data.like}
+        timeStatus={data.timeStatus}
+      />
+    </View>
+  );
+};
+
+export default class CommentList extends Component {
```
Comment (Yatanvesh) [inline src/components/Profile/CommentList.js:49]: Can be converted to functional component

File: src/components/Profile/CreatePost.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+export default class CreatePost extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      image: null,
+    };
+  }
+
+  pickImage = () => {
```
Comment (Yatanvesh) [inline src/components/Profile/CreatePost.js:86]: Extract the pick image method to utils

File: src/components/Profile/CommentCard.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
+import React, { Component } from "react";
+import { Text, StyleSheet, View, Image, TouchableOpacity } from "react-native";
+import size from "../../constants/fontSizes";
+import colors, { appTheme } from "../../constants/colors";
+
+import FontAwesome from "react-native-vector-icons/FontAwesome";
+import { spacing } from "../../constants/dimension";
+
+export default class CommentCard extends Component {
```
Comment (Yatanvesh) [inline src/components/Profile/CommentCard.js:9]: Can be converted to functional component

**Review by Yatanvesh — COMMENTED** [review]
_

File: src/components/Profile/PostCardList.js [hunk 4]
```diff
@@ -39,29 +42,80 @@ data = [
...
-        profilePicUri={data.profilePicUri}
-        title={data.title}
-        bio={data.bio}
-        discription={data.discription}
-        postPicUri={data.postPicUri}
-        like={data.like}
-        calorie={data.calorie}
-        comment={data.comment}
-        timeStatus={data.timeStatus}
+        profilePicUri={data.contentURL}
```
Comment (thirdessential) [inline src/components/Profile/PostCardList.js:51]: Revisit for correct data population

**Pull request closed without merging**